assets/datafiles/variant_map.json

```diff
@@ -0,0 +1,79 @@
+{
+  "Pumpkaboo": [
```

Comment:
Technically the app did not previously have Pumpkaboo / Gourgeist original species names at all, so these sections don't help, but to avoid special casing I decided to leave them in the file.